src/NetAnalyzers/CSharp/AnalyzerReleases.Shipped.md

```diff
@@ -0,0 +1,7 @@
+## Release 5.0
+
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+CA2014 | Reliability | Warning | CSharpDoNotUseStackallocInLoopsAnalyzer, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2014)
```

Comment:
If you define a base type `DoNotUseStackallocInLoopsAnalyzer` and have the descriptor in this base type instead of the C# specific type, then you will not need any changes to any of the Analyzer releases file. This will also ensure that all analyzer releases and released analyzers are in a single file in the core project. Would you be able to make this change?